openstack/clustering/v1/policytypes/results.go

```diff
@@ -0,0 +1,38 @@
+package policytypes
+
+import (
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// PolicyType represents a clustering policy type in the Openstack cloud
+type PolicyType struct {
+	Name          string                         `json:"name"`
+	Version       string                         `json:"version"`
+	SupportStatus map[string][]SupportStatusType `json:"support_status,omitempty"`
```

Comment:
nit: `omitempty` isn't really needed here - it's only useful in requests.

Reply:
I'll fix that too.